app/assets/scripts/views/account.js

```diff
@@ -931,30 +937,40 @@ class Account extends React.Component {
               </div>
             </div>
           </header>
-          <StickyContainer>
-            <Sticky>
-              {({ style, isSticky }) => (
-                <div style={style} className={c('inpage__nav', {'inpage__nav--sticky': isSticky})}>
-                  <div className='inner'>
-                    <ul>
-                      <li><a href='#account-information' title='Go to Operations section'>Account Information</a></li>
-                      <li><a href='#notifications' title='Go to notifications section'>Notifications</a></li>
-                      {this.isPerPermission() ? <li><a href='#per-forms' title='Go to per section'>PER forms</a></li> : null}
-                    </ul>
-                  </div>
-                </div>
-              )}
-            </Sticky>
-          </StickyContainer>
-          <div className='inpage__body'>
-            <div className='inner'>
-              {this.renderOperationsFollowing()}
-              {this.renderAccountInformation()}
-              {this.renderFieldReports()}
-              {this.props.profile.fetched && !this.props.profile.error ? this.renderSubscriptionForm() : null}
-              {this.isPerPermission() ? this.renderPerFormsComponent() : null}
+          <Tabs
+            selectedIndex={TAB_DETAILS.map(({ hash }) => hash).indexOf(this.props.location.hash)}
+            onSelect={index => this.handleTabChange(index)}
+          >
+            <TabList>
+              {TAB_DETAILS.map(tab => (
+                <Tab>{tab.title}</Tab>
+              ))}
+            </TabList>
+
+            <div className='inpage__body'>
+              <div className='inner'>
+                <TabPanel>
+                  <TabContent>
+                    {this.renderOperationsFollowing()}
+                    {this.renderAccountInformation()}
+                  </TabContent>
+                </TabPanel>
+                <TabPanel>
+                  <TabContent>
+                    {this.renderFieldReports()}
+                  </TabContent>
+                  <TabContent isError={!this.props.profile.fetched && this.props.profile.error} errorMessage="Subscriptions coming soon" title="Subscriptions">
+                    {this.renderSubscriptionForm()}
```

Comment:
Should be conditional

```suggestion
                    {this.props.profile.fetched && this.renderSubscriptionForm()}
```